autosklearn/pipeline/components/data_preprocessing/data_preprocessing_categorical.py

```diff
@@ -0,0 +1,122 @@
+import numpy as np
+
+from ConfigSpace.configuration_space import ConfigurationSpace
+
+from autosklearn.pipeline.components.data_preprocessing.imputation.categorical_imputation \
+    import CategoricalImputation
+from autosklearn.pipeline.components.data_preprocessing.one_hot_encoding.minority_coalescer \
+    import MinorityCoalescer
+from autosklearn.pipeline.components.data_preprocessing.one_hot_encoding.one_hot_encoding \
+    import OneHotEncoder
+
+
+from autosklearn.pipeline.base import BasePipeline
+
+
+class CategoricalPreprocessingPipeline(BasePipeline):
+    """This class implements a pipeline for data preprocessing of categorical features.
+    It assumes that the data to be transformed is made only of categorical features.
+    The steps of this pipeline are:
+        1 - Imputation: Make sure that categories are integers no smaller than three, and
+            assign category 2 to missing values (NaN).
+        2 - Minority coalescence: Assign category 1 to all categories whose occurence 
+            don't sum-up to a certain minimum fraction
+        3 - One hot encoding: usual sklearn one hot encoding
+
+    Parameters
+    ----------
+
+    Attributes
```

Comment:
You don't need to document attributes, see also, references and examples. Parameters are fine.

Reply:
Ok. About the parameters, I think we can talk about them in the next meeting. Because they are also not documented in other pipelines (SimpleClassificationPipeline, SimpleRegressionPipeline and BasePipeline).

Reply:
Done